tensorflow/compiler/tf2xla/kernels/reduction_ops_common.cc

```diff
@@ -111,7 +117,8 @@ void XlaReductionOp::Compile(XlaOpKernelContext* ctx) {
   xla::PrimitiveType type;
   TF_CHECK_OK(DataTypeToPrimitiveType(reduction_type_, &type));
 
-  auto data = xla::ConvertElementType(ctx->Input(0), type);
+  auto input = xla::ConvertElementType(ctx->Input(0), type);
+  auto converted_input = PreprocessInput(b, input);
```

Comment:
Sorry for the miscommunication, I meant:

```
  auto converted_input = xla::ConvertElementType(ctx->Input(0), type);
  auto preprocessed_input = PreprocessInput(b, converted_input);
```

Reply:
No worries 😃 Thanks for the review BTW.